cmd/monitoror/monitoror.go

```diff
@@ -1,37 +1,81 @@
 package main
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+	"time"
 
 	"github.com/monitoror/monitoror/cli"
+	"github.com/monitoror/monitoror/cli/commands"
+	"github.com/monitoror/monitoror/cli/helper"
+	"github.com/monitoror/monitoror/cli/version"
 	"github.com/monitoror/monitoror/config"
+	"github.com/monitoror/monitoror/registry"
 	"github.com/monitoror/monitoror/service"
+	"github.com/monitoror/monitoror/store"
 
 	"github.com/joho/godotenv"
+	"github.com/jsdidierlaurent/echo-middleware/cache"
 	"github.com/labstack/gommon/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
+func newMonitororRootCommand(monitororCli *cli.MonitororCli) {
+	cmd := &cobra.Command{
+		Use:   "monitoror",
+		Short: "Unified monitoring wallboard",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Init Service
+			server := service.Init(monitororCli.Store)
+
+			if err := helper.PrintMonitororStartupLog(monitororCli); err != nil {
+				return err
+			}
+			return server.Start()
+		},
+		Version:       fmt.Sprintf("%s, build %s", version.Version, version.GitCommit),
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+
+	cmd.Flags().BoolP("version", "v", false, "Print version information and quit")
```

Comment:
"and quit" seems unnecessary

Reply:
"Print the version number"

Reply:
"and quit" explains this argument exit monitoror, not like the other arguments

Reply:
Like all -v or --version commands? :D